$name__norm$-api/src/main/scala/$package$/api/$name__Camel$Service.scala

```diff
@@ -23,21 +29,41 @@ trait $name;format="Camel"$Service extends Service {
     */
   def useGreeting(id: String): ServiceCall[GreetingMessage, Done]
 
+
+  /**
+    * This gets published to Kafka.
+    */
+  def greetingsTopic(): Topic[GreetingMessage]
+
   override final def descriptor = {
     import Service._
     // @formatter:off
-    named("$name;format="norm"$").withCalls(
-      pathCall("/api/hello/:id", hello _),
-      pathCall("/api/hello/:id", useGreeting _)
-    ).withAutoAcl(true)
+    named("$name;format="norm"$")
+      .withCalls(
+        pathCall("/api/hello/:id", hello _),
+        pathCall("/api/hello/:id", useGreeting _)
+      )
+      .withTopics(
+        topic($name;format="Camel"$Service.TOPIC_NAME, greetingsTopic _)
+          // Kafka partitions messages, messages within the same partition will
+          // be delivered in order, to ensure that all messages for the same user
+          // go to the same partition (and hence are delivered in order with respect
+          // to that user), we configure a partition key strategy that extracts the
+          // name as the partition key.
+          .addProperty(
```

Comment:
Shouldn't we deprecate `addProperty` in favor of `withProperty` on the Scala API?